pkgs/tools/package-management/poetry/plugins/poetry-plugin-poeblix.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+}:
+
+buildPythonPackage rec {
+  pname = "poetry-plugin-poeblix";
+  version = "0.10.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "spoorn";
+    repo = "poeblix";
+    rev = "refs/tags/${version}";
+    hash = "sha256-TKadEOk9kM3ZYsQmE2ftzjHNGNKI17p0biMr+Nskigs=";
+  };
+
+  postPatch = ''
+    sed -i '/poetry =/d' pyproject.toml
+  '';
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  # infinite recursion with poetry
+  doCheck = false;
+  pythonImportsCheck = [];
```

Comment:
Why not do an import check?

Reply:
To be honest, I basically adapted the package of one of the existing poetry plugins. I just checked, and it builds fine without deactivating the checks, so I removed those lines